### PR TITLE
Add Natasha Smirnova to the student list

### PR DESCRIPTION
**Pull request by nataliasgsmirnova**
Add Natalia S to the student list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,4 @@
 | № | Имя Фамилия     | Любимый фильм                          |
 | - |:---------------:| --------------------------------------:|
 | 0 | Ава Катушка     | Три идиота                             |
+| 1 | Наташа          | Меланхолия                             |
```
Comment (ava8katushka) [inline README.md]: Please, add your surname too.

**Review by ava8katushka — CHANGES_REQUESTED** [review]
Please, address the comments and fix the conflicts.

File: README.md (hunk 1 above)
Comment (nataliasgsmirnova) [reply README.md]: updated

**Comment by ava8katushka** [comment]
Could you rename your pull request from "Update README.md" to "Add Natasha Smirnova to the student list"?

**Pull request merged**